presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -169,19 +169,20 @@ public void testRichSqlSyntax()
                 // TODO (https://github.com/prestosql/presto/issues/5837) "   CASE n_regionkey WHEN 0 THEN 'Africa' WHEN 1 THEN 'America' END region_name, \n" + // simple CASE
                 "   CASE WHEN n_name = \"BRAZIL\" THEN 'is BRAZIL' WHEN n_name = \"ALGERIA\" THEN 'is ALGERIA' ELSE \"\" END is_something,\n" + // searched CASE, double quote string literals
                 "   COALESCE(IF(n_name LIKE 'A%', NULL, n_name), 'A%') AS coalesced_name, \n" + // coalesce
+                "   round(tan(n_nationkey), 3) AS rounded_tan, \n" + // functions
                 "   `n`.`n_nationkey` + `n_nationkey` + n.n_nationkey + n_nationkey + 10000 - -1 AS arithmetic--some comment without leading space \n" +
                 "FROM `default`.`nation` AS `n`");
         assertViewQuery("" +
```

Comment:
having a separate test for each feature would be appreciated. Not it is hard to follow.

Reply:
i agree in general. I am doing this not to pay high penalty for using hive in a product test.
I think in this particular case it's a reasonable choice.